sway/commands.c

```diff
@@ -35,6 +35,8 @@
 #include "input.h"
 #include "border.h"
 
+#define ARRAY_SIZE(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
How about this: Change all definitions of struct cmd_handler arrays to:

```
static const size_t FOO_HANLDERS_SIZE = XXX;
static const struct cmd_handler foo_handlers[FOO_HANLDERS_SIZE] = {
...
};
```

This way we can replace ARRAY_SIZE with FOO_HANLDERS_SIZE and we don't have to duplicate the expansion of ARRAY_SIZE all over the place.